src/XMakeBuildEngine/Construction/ProjectElementContainer.cs

```diff
@@ -630,10 +620,7 @@ private void AddInitialChild(ProjectElement child)
 
             _count++;
 
-            if (!child.IsImplicit)
-            {
-                MarkDirty("Add child element named '{0}'", child.ElementName);
-            }
+            MarkDirty("Add child element named '{0}'", child.ElementName);
```

Comment:
This is removed because I'm reverting the original implementation where the stuff is added to the XML DOM.  There is no `IsImplicit` property anymore so the `if` is removed.

The change above this is removed for the same reason.